second.ino

```diff
@@ -315,92 +313,6 @@ int returnError(int LFSensor)
       err=6;
       break;
   }
-
```

Comment:
Don't delete this comment block yet, some conditions are missing since they were repeated and I need some words from the people who wrote it.